doc/addon.md

```diff
@@ -34,7 +34,7 @@ The `Napi::Addon<T>` class can be used together with the `NODE_API_ADDON()` and
 class ExampleAddon : public Napi::Addon<ExampleAddon> {
  public:
   ExampleAddon(Napi::Env env, Napi::Object exports) {
-    // In the constructor we declare the functions the add-on makes avaialable
+    // In the constructor we declare the functions the add-on makes available
```

Comment:
This fix is already in other PR. Could you please remove it?

Reply:
Sure thing :+1:

Reply:
@NickNaso fix removed :+1: